### PR TITLE
New footer changes

### PR DESCRIPTION
**Pull request by xtine**
Updates to new footer links and layout.
Issue: https://github.com/18F/fec-cms/issues/909
Requires: https://github.com/18F/fec-style/pull/653

Need links for:
- Privacy Policy

screen shot 2017-03-22 at 5 10 15 pm

Tablet
screen shot 2017-03-22 at 10 05 19 pm

Mobile
screencapture-127-0-0-1-8000-1490224199542

### CODE REVIEW

**Comment by jenniferthibault** [comment]
@emileighoutlaw @noahmanger is there any reason we should not link back to fec.gov for now on the following pages?
- Privacy Policy
- Plain language
- No FEAR Act

**Comment by noahmanger** [comment]
I think for No Fear and Plain Language, that works. But not sure about the Privacy Policy. We have a separate issue ( #817 ) for updating the policy to accurately describe the new site.

**Comment by emileighoutlaw** [comment]
🎉 So I updated the links (b/c we have approved pages now!), but it would be great to get a second pair of eyes on my commit

**Comment by xtine** [comment]
@emileighoutlaw: looks good! just needs the changes here as well: https://github.com/18F/openFEC-web-app/pull/1897

File: fec/fec/templates/base.html [hunk 1]
```diff
@@ -66,30 +66,80 @@
...
+              </li>
+              <li>
+                <a href="{{ cms_url }}/press/">Press</a>
+              </li>
+            </ul>
+          </div>
+
+          <div class="grid__item">
+            <ul>
+              <li>
```
Comment (noahmanger) [inline fec/fec/templates/base.html]: What about just removing this for now and we can add it once we have a link?

File: fec/fec/templates/base.html [hunk 2]
```diff
@@ -66,30 +66,80 @@
...
+
+          <div class="grid__item"></div>
+
+          <div class="grid__item">
+            <ul>
+              <li>
+                <a href="{{ cms_url }}/about/">About the FEC</a>
+              </li>
+              <li>
+                <a href="{{ cms_url }}/careers/">Careers</a>
```
Comment (noahmanger) [inline fec/fec/templates/base.html]: This should be `/about/careers/`

**Pull request merged**